services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    order = Order.objects.create(
+        user=(get_user_model().objects.get(username=username))
+    )
+    if date:
+        order.created_at = date
+        order.save()
+
+    [Ticket.objects.create(
+        movie_session=MovieSession.objects.get(pk=ticket["movie_session"]),
```

Comment:
Consider handling the case where the movie session with the given `pk` does not exist. You can use `get_object_or_404` or handle the `MovieSession.DoesNotExist` exception.